MapboxNavigation/Dictionary.swift

```diff
@@ -1,15 +1,15 @@
 import Foundation
 
-extension Dictionary where Key == Int, Value: MGLStyleValue<NSNumber> {
+extension Dictionary where Key == Int, Value: NSExpression {
     /**
      Returns a copy of the stop dictionary with each value multiplied by the given factor.
      */
     public func multiplied(by factor: Double) -> Dictionary {
-        var newCameraStop: [Int: MGLStyleValue<NSNumber>] = [:]
-        for stop in self as [Int : MGLStyleValue<NSNumber>] {
-            let f = stop.value as! MGLConstantStyleValue
-            let newValue =  f.rawValue.doubleValue * factor
-            newCameraStop[stop.key] = MGLStyleValue<NSNumber>(rawValue: NSNumber(value:newValue))
+        var newCameraStop: [Int: NSExpression] = [:]
+        for stop in self {
+            let currentValue = stop.value.constantValue as! Double
```

Comment:
Note that this will crash if the dictionary contains any expression that isn’t a constant value. Since this is a public extension on `Dictionary`, we have to assume the developer is going to try to use it wherever the compiler lets them.

I continue to believe we should make this method private or remove it outright. A few explicit multiplications won’t greatly inconvenience the developer.